site/content/en/docs/faq/_index.md

```diff
@@ -19,71 +19,74 @@ minikube start --kubernetes-version=v1.15.0
 
-First please ensure your Docker service is running and then you need to either delete the existing cluster and create one
 ```bash
 minikube delete
 minikube start --driver=docker
 ```
 
-Alternatively, if you want to keep your existing cluster you can create a second cluster with a different profile name. (example p1)
+Alternatively, (b) Create a second cluster with a different profile name:
 
 ```bash
 minikube start -p p1 --driver=docker 
 ```
 
 ## Does minikube support IPv6?
 
-minikube currently doesn't support IPv6. However, it is on the [roadmap]({{< ref "/docs/contrib/roadmap.en.md" >}}).
+minikube currently doesn't support IPv6. However, it is on the [roadmap]({{< ref "/docs/contrib/roadmap.en.md" >}}). You can also refer to the [open issue.](https://github.com/kubernetes/minikube/issues/8535)
 
 ## How can I prevent password prompts on Linux?
 
 The easiest approach is to use the `docker` driver, as the backend service always runs as `root`.
 
-`none` users may want to try `CHANGE_MINIKUBE_NONE_USER=true`,  where kubectl and such will still work: [see environment variables]({{< ref "/docs/handbook/config.md#environment-variables" >}})
+`none` users may want to try `CHANGE_MINIKUBE_NONE_USER=true`, where kubectl and such will work without `sudo`. See [environment variables]({{< ref "/docs/handbook/config.md#environment-variables" >}}) for more details.  
 
-Alternatively, configure `sudo` to never prompt for the commands issued by minikube.
+Alternatively, you can configure `sudo` to never prompt for commands issued by minikube.
 
-## How to ignore system verification?
+## How can I ignore system verification?
 
-minikube's bootstrapper, [Kubeadm](https://github.com/kubernetes/kubeadm) verifies a list of features on the host system before installing Kubernetes. in case you get this error, and you still want to try minikube anyways despite your system's limitation you can skip the verification by starting minikube with this extra option:
+[kubeadm](https://github.com/kubernetes/kubeadm), minikube's bootstrapper, verifies a list of features on the host system before installing Kubernetes. In the case you get an error and still want to try minikube despite your system's limitation, you can skip verification by starting minikube with this extra option:
 
 ```shell
 minikube start --extra-config kubeadm.ignore-preflight-errors=SystemVerification
 ```
 
-## what is the resource allocation for Knative Setup using minikube?
+## What is the minimum resource allocation necessary for a Knative setup using minikube?
 
-Please allocate sufficient resources for Knative setup using minikube, especially when you run a minikube cluster on your local machine. We recommend allocating at least 6 CPUs and 8G memory.
+Please allocate sufficient resources for Knative setup using minikube, especially when you running minikube cluster on your local machine. We recommend allocating at least 6 CPUs and 8G memory:
```

Comment:
`when you're running a minikube cluster`
or
`when running a minikube cluster`

Reply:
fixed